templates/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,15 @@
+## Description
+
+<!--- Describe your changes in detail -->
+
+Fixes: <!--- add github link -->
+
+## How Has This Been Tested?
+
+<!--- Please describe how you tested your changes. -->
```

Comment:
[ ] Please describe how you tested your changes including your testing environment?
[ ] Have the unit tests for the package passed in your environment?
[ ] If appropriate, have the regression tests affected by your code change passed?

Reply:
I think something half way in between might be better (??).  The first one asks for information and maybe isn't quite a checkbox yes/no thing.

```
<!--- Please describe how you tested your changes. -->                         

[ ] Have the unit tests for the package passed in your environment?            
[ ] If appropriate, have the regression tests affected by your code change passed?
```